### DISCUSSION TITLE
Can I somehow check how far the prerending process is and where its stuck?

### DISCUSSION THREAD

**Discussion by Niocas** [vikejs/vike#1040] [Discussion]
Can I somehow check how far the prerending process is and where its stuck? (prerendering for ~30min already)

![image](https://github.com/brillout/vite-plugin-ssr/assets/75745964/9f10f159-844a-413c-bf58-e0b5f1936685)

I added a site that applies to 160 different objects and it contains some calculations (nothing crazy),
some React States, useEffect, some input fields. After it prerenders for ~30min+ and I was wondering
if I can log somehow where it is stuck, so I can figure out what takes such a long time and improve it.

Maybe a log after page x has been rendered or something similar.

Any solutions/ideas?

**Comment by Niocas** [comment]
console.log("Something specific to site") is one solution

**Reply by brillout** [reply]
You can also add logs in `node_modules/vite-plugin-ssr/dist/cjs/node/prerender/runPrerender.js` to further narrow down the issue.

On a tangent: https://github.com/brillout/vite-plugin-ssr/issues/580.

**Reply by brillout** [reply]
Keep us updated, maybe there is something that VPS can implement to avoid / help with these kind of situations.
